src/core/server/saved_objects/saved_objects_config.ts

```diff
@@ -49,6 +49,9 @@ export const savedObjectsConfig = {
   schema: schema.object({
     maxImportPayloadBytes: schema.byteSize({ defaultValue: 26214400 }),
     maxImportExportSize: schema.byteSize({ defaultValue: 10000 }),
+    permission: schema.object({
+      enabled: schema.boolean({ defaultValue: false }),
```

Comment:
this is conflict with #5084, please make sure that who will be merged first and then rebase your branch.

Reply:
Thanks, will remove this part.

Reply:
> Thanks, will remove this part.

I assume that #5084 will merge first, then yours, right? Just want to make sure it builds and functions properly.